scripts/homework4.py

```diff
@@ -0,0 +1,125 @@
+# Author: Shad Fernandez
+# 2021-OCT-11
+# BDA 696 HW4
+
+import sys
+
+import binresponsebypredictor as bin
+import pandas as pd
+import plot_pred_response as prr
+import statsmodels.api as stats
+
+
+def main():
+    df = pd.read_csv("../datasets/Iris.csv")
```

Comment:
When initially running you assignment, there was no "Iris.csv" file in you datasets folder, so this caused an error in your code